docs/_index.md

```diff
@@ -7,7 +7,79 @@ type: "docs"
 hide_feedback: true
 ---
 
-Viam is a complete software platform for robots.
+Viam is a complete software platform for robots that runs on any 64-bit Linux OS and macOS.
+Viam supports a wide variety of systems, including:
```

Comment:
Kind of a funny distinction but do you think it might be better to say "Viam is supported on" or "Viam supports", assuming we want to make people aware that it's a "ROS replacement"? @npentrel I'm unsure.